build.yaml

```diff
@@ -0,0 +1,15 @@
+targets:
+  $default:
+    builders:
+      build_vm_compilers|entrypoint:
+        generate_for:
+          include:
+            - "test/**.dart.vm_test.dart"
```

Comment:
I know we've been using this in a lot of places, but we could probably shorten these just a bit:

For build_vm_compilers:
```
"test/**.vm_test.dart"
```

For build_web_compilers:
```
"test/**.browser_test.dart"
```